src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
```

Comment:
Using constants for health and bite values ​​will help avoid magic numbers and improve code readability.

Example:
```suggestion
const MAX_HEALTH = 100;
const BITE_DAMAGE = 50;
const DEAD_VALUE = 0;
```